pkg/restore/pod_volume_restore_action.go

```diff
@@ -260,35 +259,6 @@ func getSecurityContext(log logrus.FieldLogger, config *corev1.ConfigMap) (strin
 		config.Data["secCtx"]
 }
 
-// TODO eventually this can move to pkg/plugin/framework since it'll be used across multiple
-// plugins.
-func getPluginConfig(kind common.PluginKind, name string, client corev1client.ConfigMapInterface) (*corev1.ConfigMap, error) {
-	opts := metav1.ListOptions{
-		// velero.io/plugin-config: true
-		// velero.io/pod-volume-restore: RestoreItemAction
-		LabelSelector: fmt.Sprintf("velero.io/plugin-config,%s=%s", name, kind),
```

Comment:
Not sure if this was a bug or not, but to get unit test to pass, I had to add `=true` like [here](https://github.com/vmware-tanzu/velero/pull/6151/files#diff-bfb795c09df77918f0d92045cb9792eeae1d73051311ea3b331097398bb86981R30)

Reply:
From velero source codes, also I couldn't see any significance of this variable. But plugins also use this label-selector, so it is better to retain this variable.

Reply:
@kaovilai Sorry for my misunderstanding, you needn't add `=true`, In fact, `velero.io/plugin-config` will pattern any values with the key. No matter whether the value is empty or is true. which means that if you're using `velero.io/plugin-config`, the UT could still be passed.

But now if you change into `velero.io/plugin-config=true`, It only could pattern those resources have `velero.io/plugin-config=true`, which will make the [instruction](https://velero.io/docs/v1.11/restore-reference/#changing-pvc-selected-node:~:text=the%20second%20%22dev%22.-,Changing%20PVC%20selected%2Dnode,-Velero%20can%20update) in the doc will not work